concourse/steps/scan_sources.py

```diff
@@ -197,54 +199,99 @@ def scan_component_with_whitesource(whitesource_cfg_name: str,
                                     product_token: str,
                                     component_descriptor_path: str,
                                     extra_whitesource_config: dict,
-                                    requester_mail: str):
+                                    requester_mail: str,
+                                    cve_threshold: float,
+                                    component_name: str,
+                                    notification_recipients=[],
+                                    send_notificaton=True,):
```

Comment:
nit: please place closing paranthesis in separate line, so that adding / reordering new lines is actually more convenient